src/core/api/thread_api.cpp

```diff
@@ -202,6 +202,12 @@ otError otThreadSetNetworkName(otInstance *aInstance, const char *aNetworkName)
 
     VerifyOrExit(AsCoreType(aInstance).Get<Mle::MleRouter>().IsDisabled(), error = kErrorInvalidState);
 
+#if !OPENTHREAD_CONFIG_ALLOW_EMPTY_NETWORK_NAME
+    // Thread interfaces support a zero length name internally for backwards compatibility, but new names
+    // must be at least one valid character long.
+    VerifyOrExit(nullptr != aNetworkName && aNetworkName[0] != '\0', error = kErrorInvalidArgs);
+#endif 
```

Comment:
nit: to fix pretty check error
```suggestion
#endif
```

Reply:
Thanks for finding this quickly for me!